nodejsalert-ui/Jenkinsfile

```diff
@@ -215,11 +217,13 @@ def promoteServiceSetup(openShiftHost, openShiftToken, svcName, registry, imageN
         echo "skip dc/svc/route cleanup related exception, the resource may not exist. " + e.getMessage();
     }
     try {
-        sh """
-            oc create dc ${svcName} --image=${registry}/${imageNameSpace}/${svcName}:${tagName} -n ${projName} 2> /dev/null
+        sh """ 
+            oc create dc ${svcName} --image=${registry}/${imageNameSpace}/${svcName}:${tagName} -n ${projName} 2> /dev/null 
+            oc rollout pause dc ${svcName} -n ${projName}  2> /dev/null
+            oc patch dc ${svcName} -p '{"spec": {"template": {"spec": {"containers": [{"name": "default-container","imagePullPolicy": "Always"}]}}}}' -n ${projName} 2> /dev/null
             oc set env dc ${svcName} APP_NAME=${svcName} -n ${projName} 2> /dev/null
-            oc rollout cancel dc ${svcName} -n ${projName}  2> /dev/null
-            oc expose dc ${svcName} --port=8080 -n ${projName}  2> /dev/null
+            oc rollout resume dc ${svcName} -n ${projName}  2> /dev/null
+            oc expose dc ${svcName} --type=ClusterIP  --port=80 --protocol=TCP --target-port=8080 -n ${projName}  2> /dev/null
```

Comment:
same here port should be 8080 for nodejsalert-ui app.

Reply:
ditto

Reply:
it's fine, no change required. I thought initially that smoke test will fail for node.js app but it's passing successfully on port 8080 "http://nodejsalert-ui:8080".